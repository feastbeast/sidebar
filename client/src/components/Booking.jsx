import React from 'react';
import {Icon} from 'react-materialize';

const Booking = (props) => {

  return (
    <a href="http://www.google.com" target="_blank">
      <div className="btns"><span className="buttonText">  BOOK WITH OPENTABLE  </span><Icon className="icons bookingIcon" right>event_available</Icon></div>
    </a>
  )
}

export default Booking;
window.Booking = Booking;
