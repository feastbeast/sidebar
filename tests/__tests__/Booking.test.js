import React from 'react';
import Enzyme from 'enzyme';
import {shallow, mount, render} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Booking from '../../client/src/components/Booking';

Enzyme.configure({ adapter: new Adapter() });

describe('Booking', () => {
  it('should be defined', () => {
    expect(Booking).toBeDefined();
  });
  it('should render correctly', () => {
    const tree = shallow(
      <Booking />
    );
    expect(tree).toMatchSnapshot();
  });

});