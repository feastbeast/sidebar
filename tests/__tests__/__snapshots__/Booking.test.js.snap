// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Booking should render correctly 1`] = `
<a
  href="http://www.google.com"
  target="_blank"
>
  <div
    className="btns"
  >
    <span
      className="buttonText"
    >
        BOOK WITH OPENTABLE  
    </span>
    <Icon
      className="icons bookingIcon"
      right={true}
    >
      event_available
    </Icon>
  </div>
</a>
`;
